src/chief.t.sol

```diff
@@ -176,25 +181,36 @@ contract DSChiefTest is DSThing, DSTest {
         chief.launch();
     }
 
-    function testFail_enable_system() public {
+    function test_enable_system() public {
         address[] memory slate = new address[](1);
         slate[0] = address(0);
         gov.approve(address(chief), 100000 ether);
         chief.lock(100000 ether - 1);
         chief.vote(slate);
-        chief.launch();
+
+        assertTrue(!try_launch());
+        chief.lock(1);
+        chief.vote(slate);
+        assertTrue( try_launch());
     }
 
-    function testFail_lift_system_before_launch() public {
+    function test_lift_system_before_launch() public {
         address[] memory slate = new address[](1);
+        address    zero_slate = address(0);
```

Comment:
I'd call this `zero_address` more than `slate`. Or actually just use `address(0)` everywhere. But one single address is not a `slate`.